src/pages/index.tsx

```diff
@@ -1,7 +1,63 @@
-import React from 'react'
+import { useCallback, useEffect, useState } from 'react'
+import type { DeepReadonly, SelectValue, UseState } from '@okp4/ui'
 import type { NextPage } from 'next'
-import Home from './home/index'
+import type { Dataspace } from '../components/dataspaceDashboard/dataspaceSummary/DataspaceSummary'
+import { DataspaceSummary, DataspaceEntities, DataspaceOptions } from '../components/index'
+import './index.scss'
+import type { Config } from './api/config'
 
-const HomePage: NextPage = () => <Home />
+type HomeProps = {
+  governanceUrl: string
+}
+
+const fetchDataspace = async (url: string): Promise<Dataspace> => {
+  const response = await fetch(url)
+  const dataspace: Promise<Dataspace> = response.json()
+  return dataspace
+}
+
+const fetchGovernanceUrl = async (): Promise<string> => {
+  const response = await fetch('/api/config')
+  const config: Config = await response.json()
+  return config.app.governanceUrl
+}
+
+export const HomePage: NextPage<DeepReadonly<HomeProps>> = () => {
+  const [dataspace, setDataspace]: UseState<Dataspace | null> = useState<Dataspace | null>(null)
+  const [dataspaceGovernanceUrl, setDataspaceGovernanceUrl]: UseState<string> = useState<string>('')
```

Comment:
I will have initialized the `dataspaceGovernanceUrl` state with `null` because, by using empty strings, you take the risk to give the `DataspaceSummary` component an empty string as props (that is not good for the related button).
So, you could, as an example, disable the button if the link is `null` or `undefined`.